libparsec/crates/client/tests/unit/certif/add_device_certificate.rs

```diff
@@ -27,6 +27,27 @@ async fn ok(env: &TestbedEnv) {
     p_assert_matches!(switch, MaybeRedactedSwitch::NoSwitch);
 }
 
+#[parsec_test(testbed = "minimal")]
+async fn multiple(env: &TestbedEnv) {
+    let env = env.customize(|builder| {
+        builder.new_device("alice");
+    });
+    let alice = env.local_device("alice@dev1");
+    let ops = certificates_ops_factory(&env, &alice).await;
+
+    let switch = ops
+        .add_certificates_batch(
+            &env.get_common_certificates_signed(),
+            &[],
+            &[],
+            &Default::default(),
+        )
+        .await
+        .unwrap();
+
+    p_assert_matches!(switch, MaybeRedactedSwitch::NoSwitch);
+}
```

Comment:
I don't see where this test uses multiple certs, how multiple certs are at play here ?

Reply:
Because there is already `alice`'s device in `env`

Reply:
You mean because on line 33 we create a device alice and only line 35 we create another device alice ?

Reply:
No I mean there is already a device in the testbed `minimal`, `l.35` shows that there is one, but it's `LocalDevice`

Reply:
would be good to add a comment in the customize to explain which already existing certificates we are going to use in the integration

Reply:
Done